src/codeu/chat/server/Model.java

```diff
@@ -95,7 +95,7 @@ public Uuid userGeneration() {
   public void add(Conversation conversation) {
     conversationById.insert(conversation.id, conversation);
     conversationByTime.insert(conversation.creation, conversation);
-    conversationByText.insert(conversation.title, conversation);
+    conversationByText.insert(conversation.getTitle(), conversation);
```

Comment:
should we double check to see if there is already a conversation with the new title? I don't think the store handles duplicate keys... What happens if another conversation has the same title?

Reply:
It could either make the User choose a new title, or automatically rename the conversation to like X (2). Will work on that .